galgebra/metric.py

```diff
@@ -25,9 +25,11 @@ def apply_function_list(f,x):
 
 
 def linear_expand(expr):
+    if not isinstance(expr, Expr):
+        raise TypeError(expr + 'is not a SymPy Expr')
```

Comment:
```suggestion
        raise TypeError('{!r} is not a SymPy Expr'.fornat(expr))
```
What you have will raise TypeError in the addition.